Fibonacci.java

```
@@ -0,0 +1,18 @@
+public class Fibonacci {
+
+	public static void main(String[] args) { 
+		int nthFib = Integer.parseInt(args[0]);
```

Potential issue, need to check that the user passed a value that is parsable as an int. If they didn't you need to throw an error or exception. Please implement a test to verify what happens when an parsable string to integer is passed as cla, and when it is not parsable as an int.